### PR TITLE
Don't require the command to be the last thing in the comment

### PR DESCRIPTION
**Pull request by fiunchinho**

### CODE REVIEW

File: prow/configs.yaml [hunk 1]
```diff
@@ -304,7 +304,7 @@ presubmits:
             resources:
               requests:
                 storage: 10Mi
-    trigger: "/test create$"
+    trigger: "/test create"
```
Comment (AverageMarcus) [inline prow/configs.yaml:307]:
```suggestion
    trigger: "/test create(\s|$)"
```

File: prow/configs.yaml [hunk 2]
```diff
@@ -342,7 +342,7 @@ presubmits:
...
```
Comment (AverageMarcus) [inline prow/configs.yaml:345]:
```suggestion
    trigger: "/test create(\s|$)"
```

File: prow/configs.yaml [hunk 3]
```diff
@@ -380,7 +380,7 @@ presubmits:
...
```
Comment (AverageMarcus) [inline prow/configs.yaml:383]:
```suggestion
    trigger: "/test create(\s|$)"
```

File: prow/configs.yaml [hunk 4]
```diff
@@ -418,7 +418,7 @@ presubmits:
...
```
Comment (AverageMarcus) [inline prow/configs.yaml:421]:
```suggestion
    trigger: "/test create(\s|$)"
```

File: prow/configs.yaml [hunk 5]
```diff
@@ -456,7 +456,7 @@ presubmits:
...
```
Comment (AverageMarcus) [inline prow/configs.yaml:459]:
```suggestion
    trigger: "/test create(\s|$)"
```

File: prow/configs.yaml [hunk 6]
```diff
@@ -494,5 +494,5 @@ presubmits:
...
```
Comment (AverageMarcus) [inline prow/configs.yaml:497]:
```suggestion
    trigger: "/test create(\s|$)"
```

**Review by AverageMarcus — CHANGES_REQUESTED** [review]

File: prow/configs.yaml (hunk 6 above)
Comment (fiunchinho) [reply prow/configs.yaml:497]: What's the benefit / difference of just matching `/test create` wherever it is?

Comment (AverageMarcus) [reply prow/configs.yaml:497]: It avoid triggering on `/test created` and it allows for multiple on one line `/test create /test foobar`

**Review by AverageMarcus — APPROVED** [review]

**Pull request merged**